requirements.txt

```
@@ -1,18 +1,15 @@
 # python development requirements for the Datadog Agent
-invoke==2.2.0
 reno==3.5.0
 docker==6.0.1; python_version >= '3.7'
 docker==5.0.3; python_version < '3.7'
 docker-squash==1.0.9
 dulwich==0.20.45
-requests==2.27.1
 toml==0.10.2
 packaging==21.3
 # more recent boto3 has a bug which generates an import exception.
 # pin until this can be resolved.
 boto3==1.22.4
-botocore==1.25.4 ## for awscli
-awscli==1.23.4
+-r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/nschweitzer/requirements/requirements/common.txt
```

This should be
```suggestion
-r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/main/requirements/common.txt
```
no?